microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md

```diff
@@ -0,0 +1,112 @@
+---
+title: What's New in DevTools (Microsoft Edge 123)
+description: "Memory tool reports backing store sizes as part of an object's shallow size. Crash analyzer tool is available by default. Command Palette experimental feature has been removed. Undocked DevTools window is kept focused as the topmost window while inspecting a page. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 03/21/2024
+---
+# What's New in DevTools (Microsoft Edge 123)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Memory tool reports backing store sizes as part of an object's shallow size
+
+<!-- Subtitle: New experiment is available to make Shallow Size column in the Memory tool more helpful. -->
+
+Microsoft Edge 123 adds a new experiment called **In heap snapshots, treat backing store size as part of the containing object**.  If you enable this experiment, the **Memory** tool reports shallow sizes in a way that more closely matches JavaScript behavior.  By enabling this experiment, containers and their contents can now be represented as a single item, such as:
+* An Object and its properties.
+* An Array and its contents.
+* A Map and its contents.
+* A Set and its contents.
```

Comment:
```suggestion
* An `Object` and its properties.
* An `Array` and its contents.
* A `Map` and its contents.
* A `Set` and its contents.
```
Confirmed that they need backticks:
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

Reply:
added backticks